manager/gui/tests/unit/LrSystemHealthConfiguration.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { mount, createLocalVue } from "@vue/test-utils";
+import LrSystemHealthConfiguration from "@/components/configuration/LrSystemHealthConfiguration.vue";
+import Vuetify from "vuetify";
+import { Vue } from "vue-property-decorator";
+
+describe("LrSystemHealthConfiguration", () => {
+  let wrapper: any;
+  const localVue = createLocalVue();
+  let vuetify: Vuetify;
+
+  beforeAll(() => {
+    vuetify = new Vuetify();
+    wrapper = mount(LrSystemHealthConfiguration, { localVue, vuetify });
+  });
+
+  test(`Title should be "System Health"`, () => {
+    const pageTitle = wrapper.get(".page-title");
+    expect(pageTitle.text()).toMatch("System Health");
+  });
+
+  test(`Presence of protocol dropdown field`, () => {
```

Comment:
Presence of all the fields can be combined into one test case ? (we could have separate test cases in case if the fields are conditional hide-show or has a v-if or role based.)